Refactor what's new introduction (#1357)
mulesoft/docs-mule-runtime-3: modules/ROOT/pages/whats-new-in-mule.adoc

include::_attributes.adoc[]
endif::[]

> Mule 4.3 is the latest Mule Runtime which provides a huge performance boost on your applications, while still providing all the benefits of the Mule 4 platform that speeds up the development of applications and resilient runtime for critical mission use cases. 

Reviewer: ```suggestion
The 4.3 version of the Mule runtime engine provides a huge performance boost to your applications, while continuing to expedite application development on the Anypoint Platform and to offer a resilient runtime for use cases that are vital to the functioning your organization. 
```

Reviewer: somehow, somewhere we have to say what the Mule 4 platform is. Maybe we can link to an article on the mulesoft website? Or we have to call each of the products by their official names, which are long.

Reviewer: not quite happy with that suggestion yet either. want to clean it up a bit.

Reviewer: could say "expedite Mule application development with Anypoint Platform" 

That would avoid having to explain what the Mule 4 platform is

> It’s recommended to migrate your applications to Mule 4.3 to benefit from all the improvements that this new version provides and are explained below.

Reviewer: ```suggestion

MuleSoft recommends that you migrate your applications to Mule 4.3 to benefit from all the improvements that this new version provides.
```

include::_attributes.adoc[]
endif::[]

> The 4.3 version of the Mule runtime engine provides a huge performance boost to your applications, while continuing to expedite application development on the Anypoint Platform and to offer a resilient runtime for use cases that are vital to the functioning your organization. 

Reviewer: ```suggestion
The 4.3 version of the Mule runtime engine provides a huge performance boost to your applications, while continuing to expedite application development on Anypoint Platform and to offer a resilient runtime for use cases that are vital to the functioning your organization. 
```
